Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -337,8 +339,8 @@ def parse_response(resp, err_operation):
             return return_error(err_msg)
     # Unexpected errors (where no json object was received)
     except Exception as err:
-        err_msg = f'{err_operation}: {err}'
-        return return_error(err_msg)
+        demisto.results(f'{err_operation}: {err}')
+        sys.exit(0)
```

Comment:
is this changed intentionally? if so please explain why

Reply:
yes it is, this is one of the fixes we wanted to apply, since we dont want the costumer to get an error just a note that irt does not exist

Reply:
sys.exit(0) is not a good solution. Instead I would raise a specific exception and in the main will catch it and ignore.